Runtime/Core/SelectionPickerRenderer.cs

```diff
@@ -561,6 +561,8 @@ static ISelectionPickerRenderer pickerRenderer
 
                 go.AddComponent<MeshFilter>().sharedMesh = m;
                 go.AddComponent<MeshRenderer>().sharedMaterial = BuiltinMaterials.facePickerMaterial;
+                go.hideFlags = HideFlags.HideAndDontSave;
+                go.transform.SetParent(pb.transform.parent, false);
```

Comment:
Regarding the number of time we are using a similar type of code, Could we move the following in a new method for maintenance and future changes:
```
                GameObject go = InternalUtility.EmptyGameObjectWithTransform(pb.transform);
                go.name = pb.name + "  (Vertex Billboards)";
                go.AddComponent<MeshFilter>().sharedMesh = BuildVertexMesh(pb, map, ref index);
                go.AddComponent<MeshRenderer>().sharedMaterial = BuiltinMaterials.vertexPickerMaterial;
                go.hideFlags = HideFlags.HideAndDontSave;
                go.transform.SetParent(pb.transform.parent, false);
```

Such as `InternalUtility.MakeGameObjectWithTransform(string name, Transform t, Mesh m, Material mat)` ?

Reply:
I agree, this should have its own method at this point. Addressed this in latest changes.